src/dmd/statementsem.d

```diff
@@ -2074,6 +2091,9 @@ else
 
     override void visit(IfStatement ifs)
     {
+        /* https://dlang.org/spec/statement.html#IfStatement
+         */
```

Comment:
Do we really need to increase the height of the source code with the trailing `*/`.  Can we at least put it on one line or use the `//` comment syntax?

Reply:
I like using /* */ for comments on their own line, and // for comments appended to a line.

Reply:
The use of a block comment increases the "weight" of a comment, which is appropriate for one that is intended to document a block of code rather than a single line. // implies lightweight, line-only.